src/arrays/exercise026/Exercise026.java

```diff
@@ -0,0 +1,34 @@
+package arrays.exercise026;
+
+/***
+ * Write a Java program to move all 0's to the end of an array.
+ * Maintain the relative order of the other (non-zero) array elements.
+ */
+public class Exercise026 {
+    public static void main(String[] args) {
+        int[] arr = {1, 3, 5, 7, 0, 10, 23, 0, 32};
+        System.out.println("Original array: ");
+        for (int i : arr) {
+            System.out.print(i + "  ");
+        }
+
+        int j = 0;
+        int l = arr.length;
```

Comment:
2 zmienne, biezacyIndex, oraz indexKonca. jak zero, to stawiasz w index konca, zmiejszasz index konca